cs251tk/toolkit/__main__.py

```diff
@@ -101,15 +101,20 @@ def main():
         print('no specs loaded!')
         sys.exit(1)
 
+    available_specs = assignments
```

Comment:
Thinking about it, and I'm on my phone right now so I can't check, but since Python assigns references, I think this'll have the same issue?

I'd be more comfortable if we cloned the set - the easiest way to do that is `available_specs = set(assignments)`. 

Thoughts? Tests?

Reply:
You're right. A quick test showed me that it removes in both places.
`available_specs = set(assignments)` works as expected